pkg/runtime/runtime.go

```diff
@@ -140,8 +140,11 @@ type Route struct {
 	rules    []*runtime_pubsub.Rule
 }
 
+type TopicRoutes map[string]TopicRoute
+
 type TopicRoute struct {
-	routes map[string]Route
+	routes          Route
```

Comment:
just a comment:

this annoys me a bit. TopicRoute contains route**s** that are not actually a collection,

but it could be my lack of knowledge

Reply:
Good callout. I actually think that this can be flattened without having `route` as a struct.